compiler/backend/src/main/core/com/kitakkun/backintime/compiler/backend/transformer/implement/BackInTimeDebuggableImplementTransformer.kt

```diff
@@ -75,54 +75,44 @@ class BackInTimeDebuggableImplementTransformer : IrElementTransformerVoid() {
     private fun generateForceSetPropertyMethodBody(declaration: IrSimpleFunction): IrBody {
```

Comment:
The comment above this line is a bit outdated. Need to be fixed before merging.

Reply:
Fixed on: [Fix: outdated comment](https://github.com/kitakkun/back-in-time-plugin/pull/143/commits/8a2c7c5af284baa33839f2dddbb2a472d0df599e)